tests/testColumnOrigins.py

```diff
@@ -6,11 +6,11 @@
 from lsst.sims.catalogs.measures.instance import InstanceCatalog, cached, compound
 from lsst.sims.catalogs.generation.db import CatalogDBObject
 
-def makeTestDB(size=10, **kwargs):
+def makeTestDB(size=10, name=None, **kwargs):
```

Comment:
the default argument for name will not run, I think? so, why not make it mandatory?